.travis.yml

```diff
@@ -1,18 +1,10 @@
 ---
-# Verify this with: http://lint.travis-ci.org/
 language: ruby
 sudo: false
 # Delete dependency locks for matrix builds.
 before_install: rm Gemfile.lock || true
 script: bundle exec rake test
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.0
+  - 2.4.3
```

Comment:
Puppet 5's Ruby is 2.4.0, at least on the Xenial package I just looked at. Is there a situation where it would run under a different Ruby version?

I think it would also be worth adding a `.ruby-version` file for the rbenv users..

Reply:
The full list of which version has what is here: https://puppet.com/docs/puppet/5.5/about_agent.html

5.0.0 has ruby 2.4.1, 5.2.3 has 2.4.4

I'll push to 2.4.4 because why not